docs/content/integrations/dbt/using-dbt-with-dagster-plus.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: "Using dbt with Dagster+ | Dagster Docs"
+description: Deploy your dbt & Dagster project in Dagster+.
+---
+
+# Using dbt with Dagster+
+
+Importing an existing dbt project in Dagster+ allows you to automatically load your dbt models as Dagster assets. This can be be done with:
+
+- An existing dbt project that is not already using Dagster, or
+- A Dagster project in which your dbt project is included
+
+In this guide, we'll demonstrate by using an existing dbt project that doesn't use Dagster.
+
+---
+
+## Prerequisites
+
+To follow the steps in this guide, you'll need **Dagster+ Organization Admin**, **Admin**, or **Editor** permissions. This is required to create a code location.
+
+You'll also need **an existing dbt project** that contains the following files in the repository root:
+
+- [`dbt_project.yml`](https://docs.getdbt.com/reference/dbt_project.yml)
+- [`profiles.yml`](https://docs.getdbt.com/docs/core/connect-data-platform/profiles.yml)
+
+---
+
+## Step 1: Import your project in Dagster+
+
+<Note>
+  <strong>Heads up!</strong> This guide focuses on Serverless deployments in
+  Dagster+, but dbt projects may also be deployed if using a Hybrid deployment.
+  For more information about Hybrid deployments, refer to the{" "}
+  <a href="/dagster-plus/deployment/hybrid#hybrid-deployments-in-dagster">
+    Hybrid deployment documentation
+  </a>
+  .
+</Note>
+
+In this section, we'll demonstrate how to import an existing project Dagster+. Our example imports the project from a GitHub repository, but Dagster+ also supports Gitlab.
+
+1. Sign in to your Dagster+ account.
+
+2. Navigate to **Deployment > Code locations**.
+
+3. Click **Add code location**.
+
+4. Click **Import a dbt project**, then **Continue**.
```

Comment:
To be updated depending on previous comment

Reply:
@erinkcochran87 What do you think about updating step 4 to be a bit like the updated step 8? Something like "for dbt-only projects" and "for dbt and Dagster projects" since the import page is not the same?

Reply:
Ooh is it not? Then yes, definitely. Sorry for not remembering; I'll look at this now!

Reply:
Good catch - I forgot about this. Updated!

Reply:
By the way - in doing this, I realize these instructions don't flow as smoothly as they could due to the differences in steps between project steps. When we move our content to Markdoc, we'll be able to use partials and variables to single-source duplicate content.

By that, I mean we can create a template for reuse and use variables to do stuff like:

```
# template.md
4. Click **{{ add-project-button }}**.

# for dbt-only projects
{% include template.md add-project-button="Import a dbt project" %}

# for dbt & Dagster projects
{% include template.md add-project-button="Import a Dagster project" %}
```

Sorry for the overexplanation, just wanted it noted that we can make this better *very* soon!